extensions/azurePublish/src/index.ts

```diff
@@ -6,8 +6,8 @@ import path from 'path';
 import { v4 as uuid } from 'uuid';
```

Comment:
I highly recommend avoid using arrow functions for class methods.

Reply:
Noted. I just did minimal updates to this file, so feel free to open a PR with that change.